src/MinterGateway.sol

```diff
@@ -549,8 +552,9 @@ contract MinterGateway is IMinterGateway, ContinuousIndexing, ERC712Extended {
         // If collateral was not updated by the deadline, assume that minter's collateral is zero.
         if (block.timestamp >= collateralExpiryTimestampOf(minter_)) return 0;
 
-        uint240 totalPendingRetrievals_ = _minterStates[minter_].totalPendingRetrievals;
-        uint240 collateral_ = _minterStates[minter_].collateral;
+        MinterState storage minterState_ = _minterStates[minter_];
```

Comment:
Should be in `memory` since we don't update the state in this function.
```suggestion
        MinterState memory minterState_ = _minterStates[minter_];
```

Reply:
That is not the only consideration betweens `storage` and `memory`. If. you _will_ make changes, then you _must_ use `storage`, but if you won't make changes, then there are other considerations, and the choice is rarely memory.

Using memory here copies (SLOAD) the entire 3 slots to memory (MSTORE) only to use 2 booleans (MLOAD) to put them on the stack. Using `storage` just computes the pointer hash onto the stack, and the 2 booleans are then put on the stack as needed. The optimizer may or may not optimize that into a single SLOAD.